etn-ccis/blui-react-workflows
# Fix repsonseTime prop

Fixes 4684 .



#### Changes proposed in this Pull Request:

- Fix` responseTime` to the ForgotPasswordScreen
- 
-



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- Update code snippet `<ForgotPasswordScreen responseTime='2 business days'/>` in AppRouter.tsx.
- check http://localhost:3000/forgot-password



#### Any specific feedback you are looking for?

-
